app/src/libs/README.md

```diff
@@ -0,0 +1,5 @@
+put tun2socks.aar here.
+
+ref: 
+- https://github.com/oasiscifr/clash
+- https://github.com/oasiscifr/go-tun2socks-mobile
```

Comment:
better fork these two repositories to trojan-gfw organization

Reply:
@oasiscifr You can transfer these repos to Trojan-GFW.

Reply:
https://github.com/Dreamacro/clash/pull/341 

ing